advanced_source/custom_class_pt2.rst

```diff
@@ -0,0 +1,215 @@
+Supporting Custom C++ Classes in PyTorch 2
+==========================================
+
+This tutorial is a follow-on to the
+:doc:`custom C++ classes <torch_script_custom_classes>` tutorial, and
+introduces additional steps that are needed to support custom C++ classes in
+PyTorch 2.
+
```

Comment:
Do we have a sense of the stability of this? Does it always work now? If not, some warning about the rough edges around this (this is a "prototype" feature, right? or "unstable" in the current classification) would help